packages/grid/data-grid/src/DataGrid.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { chainPropTypes } from '@material-ui/utils';
-import { GridComponent, GridComponentProps, classnames } from '@material-ui/x-grid-modules';
+import { GridComponent,  GridComponentProps, classnames } from '../../_modules_/grid';
```

Comment:
Interesting, so that's the alternative to the replace command of https://github.com/mui-org/material-ui-x/pull/343/files#diff-1aed9b0447b7997d49dcf991dc3c4b8f. I like that these relative imports better. At least, we are upfront with it, no surprises during the build 👍 .